src/main/java/com/deavensoft/paketomat/courier/CourierServiceImpl.java

```diff
@@ -22,38 +22,39 @@ public class CourierServiceImpl implements CourierService {
     private final CenterService centerService;
     private final CityService cityService;
     private final DispatcherService dispatcherService;
-    private List<Package> packagesToDispatch = new ArrayList<>();
-    private List<City> citiesToDispatch = new ArrayList<>();
+    private final List<Package> packagesToDispatch = new ArrayList<>();
+    private final List<City> citiesToDispatch = new ArrayList<>();
 
-    public List<Courier> findAllCouriers(){
+    public List<Courier> findAllCouriers() {
         return courierRepository.findAll();
     }
 
-    public void saveCourier(Courier newCourier){
+    public void saveCourier(Courier newCourier) {
         courierRepository.save(newCourier);
     }
 
 
-    public Optional<Courier> getCourierById(Long id){
+    public Optional<Courier> getCourierById(Long id) {
         return courierRepository.findById(id);
     }
 
-    public void deleteCourierById(Long id){
+    public void deleteCourierById(Long id) {
         courierRepository.deleteById(id);
     }
+
     @Override
     public List<Package> getPackagesForCourier(String city) throws PaketomatException {
         getPackagesToDispatch();
         findCitiesInRadius(city);
```

Comment:
The same as the previous comment. The method should return a List<City>

Reply:
Done.

Reply:
Did you test changes?

Reply:
Yes i did, it is working fine.

Reply:
Yes, the application doesn't crash. But please, check the logic again. 
What list is the method supposed to return? List of all packages or list of packages to dispatch?

Reply:
Changed the list